src/black/__init__.py

```diff
@@ -368,6 +377,17 @@ def main(
     config: Optional[str],
 ) -> None:
     """The uncompromising code formatter."""
+    error_msg = "Oh no! 💥 💔 💥"
+    if revision and revision != __version__:
+        msg = (
+            f"{error_msg} The required revision `{revision}` does not match"
+            f" the running version `{__version__}`!"
```

Comment:
Half of this message says "revision" and the other half says "version" for the same concept, which might be an indication that the option name has to change :)

I don't really like the term "revision" because it makes me think of a Git commit instead of a release.

Reply:
Yep this was very awkward :D "revision" does remind me of smaller changes too, "release" sounds much better.